src/vault.md.j2

```diff
@@ -141,85 +107,84 @@ question should be able to access.
-  obtain this value is using [the KSM CLI tool](https://docs.keeper.io/secrets-manager/secrets-manager/secrets-manager-command-line-interface/init-command).
-  as described above. *This value is required.*
-
-`ksm-allow-unverified-cert`
-: Whether unverified server certificates should be accepted. If set to `true`,
-  the server certificate for connections to the KSM service will be accepted even
-  if they cannot be verified. **Unless you are a developer testing changes to
-  the KSM vault support itself, it is unlikely that you need to set this
-  property.**
-
-`ksm-api-call-interval`
-: Specify the minimum number of milliseconds between calls to the KSM API. The
-  API allows a limited number of calls per month, and calls over the included
-  amount generate additional cost. Setting this property allows you to
-  limit Guacamole's impact on that cost.
-
-`ksm-strip-windows-domains`
-: Whether or not the Windows domain should be stripped off of the username
-  when usernames are retrieved from the KSM vault and placed into its own
-  secret. This is optional, and by default it is false - domains will
-  not be stripped from the username.
-
-`ksm-allow-user-config`
-: Whether or not users should be allowed to set their own KSM configuration,
-  which will be used to pull secrets _only_ when not already provided by the
-  global or connection-group-level KSM configuration. I.E. a user-level KSM
-  configuration will never be used if a matching secret is otherwise available.
-
-#### User and Connection Group KSM Configuration
-
-In addition to the required global `ksm-config` configuration blob, Guacamole
-can also be configured with user or connection group KSM configuration, which
-will pull additional secrets _only_ when not already available. If a secret can
-be pulled using the `ksm-config` global KSM config, it will always be used.
-Failing that, if a secret is available using the connection grop config, that
-value will be used. Only when neither the global or containing connection group
-KSM configs define a secret will the user KSM config be used. Note also that
-user KSM configs will be disabled unless the global `ksm-allow-user-config` and
-per-connection `ksm-user-config-enabled` attribute are both set to true.
-
-These KSM config values can be set directly in the webapp, on the [connection
-group edit page](connection-group-management), and on the [user preferences page](preferences).
-Unlike the `ksm-config` global configuration, either the base64-encoded configuration
-provided by Keeper Commander can be used, or the one-time token can be used directly.
+All other {{ ext.properties() }} are optional.
+{% endcall %}
+
+Additional vaults for users and connection groups
+-------------------------------------------------
+
+{% call ext.config('ksm-user-vaults') %}
+In addition to the required, application-wide vault, Guacamole can be
+configured to additionally pull secrets from separate vaults that are declared
```

Comment:
Lots of addition going on, here - I feel like maybe the second instance (`additionally`) can be removed.